vectordraw/public/js/studio_edit.js

```diff
@@ -0,0 +1,139 @@
+/* Javascript for StudioEditableXBlockMixin. */
+function StudioEditableXBlockMixin(runtime, element) {
+    "use strict";
+
+    var fields = [];
+    var tinyMceAvailable = (typeof $.fn.tinymce !== 'undefined'); // Studio includes a copy of tinyMCE and its jQuery plugin
+    var errorMessage = gettext("This may be happening because of an error with our server or your internet connection. Try refreshing the page or making sure you are online.");
```

Comment:
Rather than "Try refreshing the page or making sure you are online", I would suggest "Make sure you are online, and try refreshing the page."